### ISSUE TITLE
Users are contributing

### ISSUE THREAD

**Issue by sovietspaceship2** [kelseyhightower/nocode#285]
The readme says you don't contribute; please fix the users

**Comment by harsh183** [comment]
Ah I noticed this too. I think there is a bug in the current model of the humans, infact this bug seems to have gone unnoticed for millions of years.

**Comment by daferoli** [comment]
I disagree, while the README says "you don't" for contributing, the `CONTRIBUTING.md` says you can contribute as long as it doesn't involve any code, I do see this as a discrepancy in the repo, someone should open a PR
